completions/asdf.bash

```diff
@@ -71,13 +73,13 @@ _asdf() {
     # shellcheck disable=SC2207
     COMPREPLY=($(compgen -W "--all" -- "$cur"))
     ;;
-  which)
+  which | shim-versions)
     # shellcheck disable=SC2207
-    COMPREPLY=($(compgen -c -- "$cur"))
+    COMPREPLY=($(compgen -W "$(basename "$asdf_data_dir"/shims/*)" -- "$cur"))
```

Comment:
Nice catch on these. I did a little research on `basename` and it looks like support for a `-a` flag does vary by system, so to be safe I've used a loop. I've also fixed the empty glob issue by enabling the `nullglob` option (in a subshell to avoid contaminating the users environment).